_pkgdown.yml

```diff
@@ -9,7 +9,7 @@ development:
 
 navbar:
   structure:
-    left:  [home, introduction, reference, articles, news, benchmarks]
```

Comment:
I think we should not rename articles to vignettes, pkgdown is sensitive to those names, they has to be hardcoded somewhere. You can observe now that on website we still have `references` tab despite you have removed it. It seems to be added anyway. I tried to rename it to "manual", then it was duplicated, one from "manual" and another added automatically "reference". The proper place for translation of labels seems to be in the "components" section, and here in "structure" we should align more to pkgdown terminology.